index.html

```diff
@@ -0,0 +1,178 @@
+                    <li><i class="fas fa-wifi"></i></li>
+                    <li><i class="fas fa-battery-full"></i></li>
+                </ul>
+            </div>
+            <i class="fas fa-times cancel"></i>
+            <ul class="menu-list">
+                <li><div class="box"><a href="#work">Portfolio</a></div></li>
+                <li><div class="box"><a href="#about">About</a></div></li>
+                <li><div class="box"><a href="#contact">Contact</a></div></li>
+            </ul>
+        </div> 
+        <div class="head">
+            <div class="header-headings">
+                <h3 class="header-one">An Online Education for All!</h3>
+                <h1 class="header-two">MASSIVE OPEN ONLINE COURSE SUMMIT</h1>
+            </div>
+            <div class="header-para">
+                <p>A Global online conference that discusses all possible approaches to adopt for the Online Education Program.</p>
+            </div>
+            <div class="header-cont">
+                <h2 class="para-time">2022.04.15(Thurs) ~ 16(Fri)</h2>
+                <p class="para-contact">@globalonlineprogram.com</p>
+            </div>   
+        </div>
+        <section class="main">
+            <h3 class="program">Main Program</h3>
+            <div class="big-forum">
+                <div class="forum">
+                    <i class="fas fa-laptop"></i>
+                    <h3>Lecture</h3>
+                    <p>listen to various speakers introducing what MOOC stands for</p>
+                </div>   
+                <div class="forum">
+                    <i class="fas fa-code"></i>
+                    <h3>Approaches</h3>
+                    <p>Challenges for the Online Education</p>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-atom"></i>
+                    <h3>Forum</h3>
+                    <P>Have time to share your thoughts and opinions with experts.</P>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-code"></i>
+                    <h3>Status</h3>
+                    <P>Different online Educational platforms</P>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-database"></i>
+                    <h3>Workshop</h3>
+                    <P>Collection of all Questions and Answers</p>
+                </div>
+
+            </div>
+            <button type="button" class="join">Join the MOOC Summit 2022</button>
+                            
+        </section>
+        <section class="speakers">
+            <!-- <div class="speaker"> -->
+                <!-- <img src="mooc-images/speaker1.png" alt="The first speaker "> -->
+                <!-- <div class="him"> -->
+                    <!-- <h4>Yochai Benkler</h4> -->
+                    <!-- <p class="p"><em> Professor and entrepreneur and Legal Studies at Stanford University</em></p> -->
```

Comment:
- [ ] I recommend deleting the unused and commented-out code because this makes your codebase unclean.